src/passes/Asyncify.cpp

```diff
@@ -549,7 +549,8 @@ class ModuleAnalyzer {
                             return !info.isBottomMostRuntime &&
                                    !info.inBlacklist;
                           },
-                          [](Info& info) { info.canChangeState = true; });
+                          [](Info& info) { info.canChangeState = true; },
+                          scanner.IgnoreIndirectCalls);
```

Comment:
It seems strange to use `scanner` in the path to `IgnoreIndirectCalls`, but I guess the enclosing type is a pain to type out, so this is fine.

Reply:
Yeah, this seemed shorter, but I'm not totally happy with it either. Another option might be `get_type_or_whatsitcalled(scanner)::IgnoreIndirectCalls`?